torch/csrc/jit/passes/onnx/scalar_type_analysis.cpp

```diff
@@ -292,7 +292,7 @@ static void UpdateScalarTypeForInputs(
         const_node->t_(attr::value, new_val);
         const_node->insertBefore(n);
         const_node->output()->setType(TensorType::create(new_val));
-        const_node->copyMetadata(input->node());
```

Comment:
Maybe a comment here to hint future readers?

Reply:
This lines looks the same as 291. Just for my learning - why do we do it twice?

Reply:
+1 copyMetadata is a nightmare to me. I see them everywhere and not exactly know what they contribute ...

Reply:
> +1 copyMetadata is a nightmare to me. I see them everywhere and not exactly know what they contribute ...

lol I often need to check its source code to confirm. The name is too general. Probably a good idea to add some comments. 

> This lines looks the same as 291. Just for my learning - why do we do it twice?

My guess is unfound merge 'conflict'.